components/compliance-service/api/reporting/server/server.go

```diff
@@ -914,3 +915,40 @@ func (srv *Server) GetReportContent(ctx context.Context, in *reporting.ReportCon
 	}
 	return nil*/
 }
+func (srv *Server)AssetCount(ctx context.Context , in *reporting.ListFilters) (*reporting.AssetSummary , error) {
+	formatttedFilters := formatFilters(in.Filters)
+	var count reporting.AssetSummary
+	if len(formatttedFilters["start_time"]) <= 0 || len(formatttedFilters["end_time"]) <= 0 {
+		return nil , &errorutils.InvalidError{}
+	} else if reflect.DeepEqual(formatttedFilters["start_time"] , formatttedFilters["end_time"]) {
+		return nil , &errorutils.InvalidError{}
+	} else {
```

Comment:
As previous conditions are returning, this else isn't required,  remove the else condition and put the block statemnt below line 925